neurom/utils.py

```diff
@@ -39,9 +39,9 @@
 
 def warn_deprecated(msg):
     """Issue a deprecation warning."""
-    warnings.simplefilter('always', DeprecationWarning)
-    warnings.warn(msg, category=DeprecationWarning, stacklevel=2)
-    warnings.simplefilter('default', DeprecationWarning)
+    with warnings.catch_warnings():
+        warnings.simplefilter("always", DeprecationWarning)
```

Comment:
Isn't `"once"` enough? `"always"` might spam the logs a bit too much, isn't it?

Reply:
Yeah, good point.

Reply:
I changed it but it doesn't seem to have an effect. I believe it's because of the context manager resetting the configuration at each call.